env.example

```diff
@@ -1,8 +1,15 @@
-export LOCATION=eastus
+export LOCATION=westeurope
 export ARO_IMAGE=arointsvc.azurecr.io/aro:latest
 export NO_CACHE=false
 export AZURE_EXTENSION_DEV_SOURCES="$(pwd)/python"
 
 export ARO_SKIP_PKI_TESTS=true
 
+export OPENSHIFT_VERSION="4.13.23"
+export CLUSTER_RESOURCEGROUP="${USER}-v4-$LOCATION"
+export CLUSTER_NAME="${USER}-aro-cluster"
+export ARO_ADOPT_BY_HIVE="true"
+export ARO_INSTALL_VIA_HIVE="true"
+export HIVE_KUBE_CONFIG_PATH="aks.kubeconfig"
```

Comment:
comment: My suspicion is that we don't need these locally, because they are hard-coded and set at the container level with `-e` arguments.